tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -1,8 +1,8 @@
 tests:
   kernel.timer:
-    tags: kernel
+    tags: kernel flarp
```

Comment:
What's a flarp?

Reply:
Yeah, I just noticed that.  I was just using that to easily filter tests locally.  Sigh.